tests/test_monitor.py

```diff
@@ -94,20 +108,29 @@ def test_basic_monitor(monitor, tn_client, loop):
     assert 'Task' in resp
 
     resp = execute(tn, 'ps 123\n')
-    assert 'Task' in resp
+    assert 'TypeError' in resp
 
     resp = execute(tn, 'signal name\n')
     assert 'Unknown signal name' in resp
 
     resp = execute(tn, 'stacktrace\n')
     assert 'loop.run_forever()' in resp
 
-    resp = execute(tn, 'wehere 123\n')
```

Comment:
🤦‍♂️ why this event worked?

Reply:
Why it worked in the past you mean? Because with the old code any command beginning with a "w" would be routed to "where".

Reply:
yeah, stupid error on my side :)

Reply:
It did take me a while to realise what was going on as well :D